examples/js/exporters/GLTFExporter.js

```diff
@@ -459,21 +473,36 @@ THREE.GLTFExporter.prototype = {
 			var mimeType = map.format === THREE.RGBAFormat ? 'image/png' : 'image/jpeg';
 			var gltfImage = { mimeType: mimeType };
 
-			if ( options.embedImages ) {
+			if ( options.embedImages || ( options.forcePowerOfTwoTexture && ! isPowerOfTwo( map.image ) ) ) {
 
 				var canvas = cachedCanvas = cachedCanvas || document.createElement( 'canvas' );
+
 				canvas.width = map.image.width;
 				canvas.height = map.image.height;
+
+				if ( options.forcePowerOfTwoTexture ) {
+
+					if ( ! options.embedImages ) {
+
```

Comment:
I don't understand this check. We're showing the warning only if `embedImages` is `false`?

Reply:
In that case, user doesn't set `embedImages` true so expects non-embedded image., but image will be embedded so I wanna notify it to users.